### PR TITLE
Removing noMobileAds switch

### PR DESCRIPTION
**Pull request by uplne**
1. this PR removes the switch and makes it a default behaviour
2. as we don't have 300x50 anymore this also sets default height to ad slot to 250px

@kelvin-chappell @jbreckmckye @Calanthe

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js [hunk 1]
```diff
@@ -114,7 +110,7 @@ define([
                 inlineMercPromise = Promise.resolve(null);
             }
 
-            if (config.switches.viewability && !inMobileAdsTest()) {
+            if (config.switches.viewability) {
```
Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js]: if viewability is on and breakpoint is mobile should this apply?  I don't know.

**Comment by kelvin-chappell** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js (hunk 1 above)
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js]: Nice catch. We need the mobile check for this. I've also included a few new unit tests so it should be nicely covered now.

**Comment by uplne** [comment]
I am closing this one. There were some changes to the switches in the mean time. I am ending up with a lot of 'merge master' commits and I need to test it one more time properly.

I will extend the switch and do it one more time this week.

**Pull request closed without merging**